file_paths.md

```diff
@@ -163,7 +479,7 @@ The files annotated by an '*' indicate that they are created by package installa
         mcollectived.pid
         pxp-agent.pid
 
-# puppet-agent (windows)
+# puppet-agent 4, 5 (Windows)
 
 On recent versions of Windows, e.g. 2008 & 2012, the installation path
```

Comment:
nit: reword `On recent versions of Windows, e.g. 2008 & 2012`  2008 is hardly recent.  In fact we should probably remove this as the only applicable OSes this applies to are no longer supported by Microsoft or Puppet